pages/1.13/security/oss/managing-users/external-users/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: layout.pug
+navigationTitle: External users
+title: External users
+excerpt: Managing external DC/OS users
+menuWeight: 10
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+An external user is automatically created for the first user who logs in to the DC/OS cluster via a single sign-on flow through Auth0.
```

Comment:
"An external user account is...".

Remove "through Auth0" - it doesn't add much here.